test/config.test.ts

```diff
@@ -23,9 +23,18 @@ test("configKeysInputValidation", () => {
   const validObjectRes = configKeysInputValidation.validate('{"key_1": "First value", "key_2": "Second value"}');
   expect(validObjectRes.ok && validObjectRes.value).toStrictEqual({ key_1: "First value", key_2: "Second value" });
 
+  const validConversionsRes = configKeysInputValidation.validate(
+    '{"key_1": "First value", "key_2": 42, "key_3": false}'
+  );
+  expect(validConversionsRes.ok && validConversionsRes.value).toStrictEqual({
+    key_1: "First value",
+    key_2: "42",
+    key_3: "false"
```

Comment:
It stills output strings even though it validates numbers/boolean? oO

Reply:
Mentioned it this morning, but for the record:
- our API expects only strings for sys props/env vars
- but it's nicer to the users to let them use `42` rather than `"42"` when they configure, e.g., a duration, number of users, whatever.